src/dev-app/BUILD.bazel

```diff
@@ -111,8 +113,8 @@ expand_template(
     template = "system-config-tmpl.js",
 )
 
-dev_server(
-    name = "devserver",
+filegroup(
```

Comment:
Add a comment that explains that the filegroup is used both for running the devapp and for outputting files for firebase deployment?

Reply:
done.